### ISSUE TITLE
During Fine Tune Turkish Language not decrease Loss Metrics

### ISSUE THREAD

**Issue by Yaltay** [OpenMOSS/MOSS-TTS-Nano#57]
I'm Fine tuneing Turkish language model , but why cant  reduce them during the FT process?
This Ft metrics 
--per-device-batch-size 8   --gradient-accumulation-steps 1 learning-rate 5e-6   --warmup-ratio 0.03   --num-epochs 25   --mixed-precision bf16 --weight-decay 0.01  --max-length 1024   --channelwise-loss-weight 1,48

**Comment by alpacaking** [comment]
Thanks for reporting this.

For adapting to a new language such as Turkish, `5e-6` may be too conservative. You can try a slightly higher learning rate, for example `1e-5` first, and `5e-5` if the loss is still almost flat.

Could you share a few examples from your raw training data before preprocessing? A few JSONL lines would be enough, including fields such as `text`, `audio`, and `ref_audio` if reference audio is used.

This would help us check whether the data format and text/audio setup look normal.

**Comment by Yaltay** [comment]
Thanks for the response. @alpacaking 

Here is the audio length distribution of the dataset:

* 2–4s: 736 samples (2%)
* 4–6s: 1,404 samples (3%)
* 6–8s: 1,438 samples (4%)
* 8–10s: 1,588 samples (4%)
* 10–12s: 1,861 samples (5%)
* 12–15s: 3,377 samples (8%)
* 15–20s: 8,742 samples (21%)
* 20–30s: 21,964 samples (53%)

Most of the samples are between 15 and 30 seconds.

Additionally, here are a few example lines from the JSONL file:

```json
{"audio": "/workspace/turkish_data/clips/fMhCAO9R3H0/0000.wav", "text": "Bugün, ee, belki de sıkça duyduğumuz ama sonuçlarını pek de düşünmediğimiz bir konuya, yani banka hesabımızı başkasına kullandırma meselesine eğiliyoruz. Çok masum bir yardım gibi başlayabilir, değil mi? Ama işin aslı pek öyle olmayabilir.", "ref_audio": "/workspace/turkish_data/clips/fMhCAO9R3H0/0001.wav", "language": "tr"}

{"audio": "/workspace/turkish_data/clips/fMhCAO9R3H0/0001.wav", "text": "Evet, yani bu durumun hukuki ve teknik boyutları aslında oldukça derin. Elimizdeki temel kaynakta adli bilişim ve siber güvenlik uzmanı Cüneyt Üren'in hazırladığı gerçekten kapsamlı bir inceleme yazısı.", "ref_audio": "/workspace/turkish_data/clips/fMhCAO9R3H0/0000.wav", "language": "tr"}
```

**Comment by alpacaking** [comment]
The raw training data format looks normal to me. For Turkish adaptation, you may continue trying a higher learning rate.

**Comment by 1PD-IS-NO-1** [comment]
Hi Dear ,
          can you tell me by fine tuning can we add an Indian language like Hindi.
How many hours of data will be enough to get a good quality audio from tts in hindi. (moss-tts-nano 100M)

like okay maybe you don't know about hindi language but can you tell me apart from those already existing 20 languages  can we add a new language in moss-tts-nano(100m).

I will appreciate your response because it will save my precious time dear.
